core/01_core.js

```diff
@@ -479,6 +480,11 @@
     };
   }
 
+  // Default impl of contextual logging
+  op_get_ext_import_meta_proto().log = function internalLog(level, ...args) {
+    op_print(`[${level.toUpperCase()}] ${consoleStringify(...args)}\n`, false);
```

Comment:
I think this should log to stderr instead of stdout? Maybe this should be `logError`?

Reply:
technically it depends on the `level`. but directly above this there is a default console implementation that always calls `op_print` so i think this is fine.

Reply:
Ref: https://github.com/denoland/deno/blob/60b502db80ea4524c8a5e316177f32329dc15a33/ext/http/00_serve.ts#L824